ssas/systems.go

```diff
@@ -82,9 +82,9 @@ type ClientToken struct {
 	gorm.Model
 	Label     string    `json:"label"`
 	Uuid      string    `json:"uuid"`
-	System    System    `gorm:"foreignkey:SystemID;association_foreignkey:ID"`
-	SystemID  uint      `json:"system_id"`
-	ExpiresAt time.Time `json:"expires_at"`
+	System    System    `gorm:"foreignkey:SystemID;association_foreignkey:ID" json:"-"`
+	SystemID  uint      `json:"-"`
+	ExpiresAt time.Time `json:"-"`
```

Comment:
we don't want to expose when it expires? Is that for security purposes? It seems like it could be helpful information.

Reply:
@MrBilnon care to add client token expiration date?

Reply:
dobe